showcase/src/assets/translations/en.json

```diff
@@ -330,6 +330,7 @@
         },
         "REGULAR": "Regular Table",
         "WITH_SELECTION": "Table with selection",
+        "WITH_ITEMS_PER_PAGE_SELECTOR": "Table with custom number of items per page",
```

Comment:
I suggest to change this for:
"Table with the selector of items per page to display"